src/main/java/io/nats/client/Options.java

```diff
@@ -1731,13 +1771,8 @@ public Builder(Options o) {
     // CONSTRUCTOR
     // ----------------------------------------------------------------------------------------------------
     private Options(Builder b) {
-        if (b.natsServerUris.isEmpty()) {
-            this.natsServerUris = Collections.singletonList(DEFAULT_NATS_URI);
-        }
-        else {
-            this.natsServerUris = Collections.unmodifiableList(b.natsServerUris);
-        }
-        this.unprocessedServers = b.unprocessedServers;  // exactly how the user gave them
+        this.natsServerUris = Collections.unmodifiableList(b.natsServerUris);
+        this.unprocessedServers = Collections.unmodifiableList(b.unprocessedServers);  // exactly how the user gave them

```

Comment:
1.After reviewing the builder, I noticed that `natsServerUris` was already never empty so that was unnecessary. 
2. All lists, since they can be viewed from public api needs to be unmodifiable.